components/net_traits/request.rs

```diff
@@ -106,6 +107,25 @@ pub enum CORSSettings {
     UseCredentials
 }
 
+pub trait FetchTaskTarget {
```

Comment:
I would arguably put this in `net_traits/lib.rs`, `use net_traits::request::FetchTaskTarget` doesn't quite appeal to me.